src/main/java/edu/cornell/kfs/concur/service/impl/ConcurAccountValidationServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package edu.cornell.kfs.concur.service.impl;
+
+import java.util.ArrayList;
+
+import edu.cornell.kfs.concur.businessobjects.ConcurAccountInfo;
+import edu.cornell.kfs.concur.businessobjects.ValidationResult;
+import edu.cornell.kfs.concur.service.ConcurAccountValidationService;
+
+public class ConcurAccountValidationServiceImpl implements ConcurAccountValidationService {
+
+    @Override
+    public ValidationResult validateConcurAccountInfo(ConcurAccountInfo concurAccountInfo) {
+        ValidationResult validationResult = new ValidationResult(false, new ArrayList<String>());
```

Comment:
yes, current account validation cannot be used. A new service will be implemented to do account validation. I will be working on that as a separate story.